tests/karma/unit/enketo/phone-widget.js

```diff
@@ -0,0 +1,116 @@
+describe('phone-widget', function() {
```

Comment:
PR at https://github.com/medic/medic-webapp/pull/1791 to revert to previous style